src/components/Avatar.jsx

```diff
@@ -12,10 +12,10 @@ const makeStyle = ({ faded, color }) => {
     : { backgroundColor: color }
 }
 
-const Avatar = ({ icon, color, faded }) => {
+const Avatar = ({ icon, color, faded, ghost }) => {
   const style = useMemo(() => makeStyle({ faded, color }), [color, faded])
```

Comment:
is makeStyle a costy operation?

Reply:
yup, c'est même un souci remonté par Mui (j'ai plus la ref sous la main par contre) - enfin si on utilise ici le makeStyle de Mui... 🤔